.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null
+}
```

Comment:
What is this for? @SaiVarunVaranasi

Reply:
That got added automatically. I didn't make any changes to that file.

Reply:
okay ^^